src/Storefront/Resources/app/storefront/src/scss/component/_image-slider.scss

```diff
@@ -29,7 +29,6 @@ based on "base-slider" component and "tiny-slider" (https://github.com/ganlanyua
 
             .image-slider-image {
                 object-fit: cover;
-                font-family: 'object-fit: cover;'; // IE polyfill
```

Comment:
We will remove the IE compatibility only with the next major version, but currently IE is still supported for now (at least for the storefront part)